.github/workflows/packaging.yml

```diff
@@ -81,6 +81,6 @@ jobs:
             CURL_CMD+=" -F $(basename ${f})=@./build/${f}"
           done
 
-          echo ${CURL_CMD}
+          echo "Upload packages..."
```

Comment:
I would keep echoing the command, but without actual username/password. It may be useful for debugging.

How about something like the following?

```bash
CURL_CMD=<...>
CURL_CMD_ECHO=<...>

for <...>; do
    CURL_CMD=<...>
    CURL_CMD_ECHO=<...>
done

echo ${CURL_CMD_ECHO}
${CURL_CMD}
```

Sooner or later this come should go into rws'es GitHub action, so some bloating is not critical.